testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -53,12 +51,10 @@ def convert_aa_code_long_data():
         ('Q', ('GLN',)),
         ('C', ('CYS', 'CYSH', 'CYS1', 'CYS2')),
     ]
-    data = []
     for resname1, strings in aa:
         for resname3 in strings:
-            data.append((resname3, resname1))
+            yield (resname3, resname1)
```

Comment:
No, this isn't a yield based test. @jbarnoud recommended this https://github.com/MDAnalysis/mdanalysis/pull/1494#discussion_r129864020